### PR TITLE
base64: add Turkish translation

### PR DESCRIPTION
**Pull request by Yutyo**
- [x] The page (if new), does not already exist in the repo.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/master/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages.tr/common/base64.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# base64
+
+> Bir dosya veya standart veriyi Base64 formatında şifrele veya yalın veri çıktısı olarak deşifre et.
```
Comment (bl-ue) [inline pages.tr/common/base64.md:3]: There should be an empty newline after the page description.
```suggestion
> Bir dosya veya standart veriyi Base64 formatında şifrele veya yalın veri çıktısı olarak deşifre et.

```

**Review by bl-ue — CHANGES_REQUESTED** [review]
Looks good. I'd like to wait on merging this, though, because we have a PR in progress (#5234) that's modifying `base64.md` and `base32.md` and I'd like to get that merged and this updated first.

File: pages.tr/common/base64.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+- Bir dosyayı şifrele:
+
+`base64 {{dosyaismi}}`
+
+- Bir dosyayı deşifre et:
+
+`base64 -d {{dosyaismi}}`
```
Comment (bl-ue) [inline pages.tr/common/base64.md:11]:
```suggestion
`base64 --decode {{dosyaismi}}`
```

File: pages.tr/common/base64.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+- stdin'den şifrele:
+
+`{{herhangibirkomut}} | base64`
+
+- stdin'den deşifre et:
+
+`{{herhangibirkomut}} | base64 -d`
```
Comment (bl-ue) [inline pages.tr/common/base64.md:19]:
```suggestion
`{{herhangibirkomut}} | base64 --decode`
```

**Review by bl-ue — CHANGES_REQUESTED** [review]
@Yutyo, please apply these changes from #5234.

**Review by bl-ue — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @Yutyo!

**Pull request merged**